iree/compiler/Dialect/VM/Target/C/TranslationFlags.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef IREE_COMPILER_DIALECT_VM_TARGET_C_TRANSLATIONFLAGS_H_
+#define IREE_COMPILER_DIALECT_VM_TARGET_C_TRANSLATIONFLAGS_H_
+
+#include "iree/compiler/Dialect/VM/Target/C/CModuleTarget.h"
+
+namespace mlir {
+namespace iree_compiler {
+namespace IREE {
+namespace VM {
+
+// Returns a CCodeTargetOptions struct initialized with the
+// --iree-vm-c-* flags.
+CCodeTargetOptions getCCodeTargetOptionsFromFlags();
```

Comment:
I would suggest to rename this to `CModuleTargetOptions` or even `CTargetOptions` as already discussed :)